code/src/App.js

```diff
@@ -1,9 +1,85 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Summary } from './Summary.js'
+import { Happiness } from './Happiness'
+import { Input } from './Input'
+import { Dropdown } from './Dropdown'
 
 export const App = () => {
+  const [happiness, setHappiness] = useState('')
+  const [feeling, setFeeling] = useState('')
+  const [showResult, setShowResult] = useState(false)
+  const [question, setQuestion] = useState('feeling')
+  const [treatment, setTreatment] = useState("");
+  
+
+  const handleSubmit = (event) => {
+   event.preventDefault() //spara informationen så att sidan inte laddas om (det vanliga sidobeteendet)
+    setShowResult(true)
+  }
+
+  const handleContinueClick = () => {
+    console.log('click continue')
+    if (question === 'feeling') {
+      setQuestion('happiness') 
+    } else if (question === 'happiness') {
+        setQuestion('treatment')
+      
+    }
+  }
```

Comment:
Since it's possible to submit the form already on question 1 when using the enter key on your keyboard you could bake this into one function.
something like:
```
 const handleSubmitOrContinue = (event) => {
   event.preventDefault()

    if (question === 'feeling') {
      setQuestion('happiness') 
    } else if (question === 'happiness') {
        setQuestion('treatment')
    } else {
       // do the actualy submit.
       setShowResult(true)
    }
  }
```